web/html/src/branding/css/susemanager/components/date-time-picker.less

```diff
@@ -1,6 +1,6 @@
 /* This element already has `position: absolute` */
 .react-datepicker-popper {
-  z-index: 1010 !important;
+  z-index: 1050 !important;
```

Comment:
There is any convention on the values used in the z-index?
Why we go from 1 to 1010, to 1050, and so on.
Why we don't use 1,2,3,4,... ?

Reply:
@srbarrios There's a few issues we're solving here, the main one is that most indexes don't actually come from our code, but instead from third-party dependencies and we have no control over the values they use. So we may have values from one dependency that are 1, 2, 3 and from another dependency that are 1000, 1500, 2000 etc. In our theme we want to mostly not override everything because then things tend to break when we update those dependencies. Hence the odd numbers.

The other problem we're addressing is adding new items to the stacking order after the fact. If we have say an order of 1, 2, 3, 4, 5 and we want to add an item between 2 and 3 we have to go and also change the definitions for 3, 4 and 5. If we instead have values such as 10, 20, 30, 40, 50 and we want to put an item between 20 and 30 we can add a new item with a value of 25 and not change any other code. This allows us to add quick fixes such as this without making big changes and iron out any weird values whenever we do some bigger refactoring.

Hope this answers your question.